doc/hacking/using-docker.rst

```diff
@@ -41,6 +41,10 @@ the first time is by running these commands::
          coffee (and maybe redecorate your living room), as the initial build
          may take a while.
 
+.. warning:: Running ``docker compose up`` will take over the local ports 8080
+             (for the website) abd 8000 (for graphite). You can override this
+             with a ``docker-compose.override.yml`` file.
+
```

Comment:
Actually, `docker-compose.yml` will bind to port 80 on the local system, not 8080.